library/kani/src/vec.rs

```diff
@@ -0,0 +1,12 @@
+// Copyright Kani Contributors
+// SPDX-License-Identifier: Apache-2.0 OR MIT
+use crate::{any, Invariant};
+
+/// Generates an arbitrary vector given a length.
+pub fn any_vec<const VEC_LENGTH: usize, T>() -> Vec<T>
+where
+    T: Invariant,
```

Comment:
`T: Arbitrary`

Reply:
Yes, I'll change to `T: Arbitrary` when resolving the max size issue. 

I'm curious why any only requires Invariant?

Reply:
Update: I'm going to have to go with  `T: Invariant` since I want to be minimal diff between high-performing `exact_vec` and not-so-well `any_vec` and `exact_vec` needs `T: Invariant`.

If the performance issue is fixed (I'm asking @tautschnig tomorrow 6-30), then we can consider ways to loosen it.